sandbox/src/main/java/com/badoo/ribs/sandbox/app/RecyclerViewTestActivity.kt

```diff
@@ -72,8 +72,9 @@ class RecyclerViewTestActivity : RibActivity() {
         SwitcherBuilder(
             object : Switcher.Dependency {
                 override val activityStarter: ActivityStarter = integrationPoint.activityStarter
-                override val permissionRequester: PermissionRequester = integrationPoint.permissionRequester
-                override val dialogLauncher: DialogLauncher = integrationPoint
+                override val permissionRequester: PermissionRequester =
+                    integrationPoint.permissionRequester
+                override val dialogLauncher: DialogLauncher = integrationPoint.dialogLauncher
```

Comment:
Why we need PermissionRequester as well as DialogLauncher as part of dependency now? Node have it under the hood

Reply:
Intended to be done in a separate PR.